blockonomics-woocommerce.php

```diff
@@ -409,22 +409,37 @@ function show_basic() {
     function bnomics_display_tx_info($order, $email=false)
     {
         $blockonomics = new Blockonomics();
-        $active_cryptos = $blockonomics->getActiveCurrencies();
-        foreach ($active_cryptos as $crypto) {
-            $txid = get_post_meta($order->get_id(), 'blockonomics_'.$crypto['code'].'_txid', true);
-            $address = get_post_meta($order->get_id(), $crypto['code'].'_address', true);
-            if ($txid && $address) {
-                if ($crypto['code'] == 'btc') {
-                    $base_url = Blockonomics::BASE_URL;
-                }else{
-                    $base_url = Blockonomics::BCH_BASE_URL;
+        $txid = get_post_meta($order->get_id(), 'blockonomics_payments_txids', true);
+        $address = get_post_meta($order->get_id(), 'blockonomics_payments_addresses', true);
+
+        if ($txid && $address) {
+            $txidArray = explode(",", $txid);
+            $addressdArray = explode(",", $address);
+            if (!empty($txidArray)) {
+                echo '<b>'.__('Payment Details', 'blockonomics-bitcoin-payments').'</b><p><strong>'.__('Transaction', 'blockonomics-bitcoin-payments').':</strong>';
+                    
+                for ($i = 0; $i < count($txidArray); $i++) {
```

Comment:
We can move `count($txidArray)` out of the for loop, as this is called on each iteration and also reused later to check if it is not the last iteration.
```
$txidCount = count($txidArray);
for ($i = 0; $i < $txidCount ; $i++) {
    ....
    if ($i < $txidCount - 1) {
        echo ', ';
    }
}